x/crosschain/keeper/msg_server_migrate_tss_funds_test.go

```diff
@@ -35,20 +35,38 @@ func TestMsgServer_MigrateTssFunds(t *testing.T) {
 		_, found := k.GetCrossChainTx(ctx, index)
 		assert.True(t, found)
 	})
+	t.Run("not enough funds in tss address for migration", func(t *testing.T) {
```

Comment:
do you mean "not enough funds to initiate tss migration cctx" ?

Reply:
yeah